src/python/ensembl/io/genomio/gff3/process.py

```diff
@@ -441,7 +444,7 @@ def _normalize_transcripts(self, gene: SeqFeature, fail_types) -> SeqFeature:
                 message = (
                     f"Unrecognized transcript type: {transcript.type}" f" for {transcript.id} ({gene.id})"
                 )
-                print(message)
+                logging.warning(message)
```

Comment:
```suggestion
                logging.warning(
                    f"Unrecognized transcript type: {transcript.type}" f" for {transcript.id} ({gene.id})"
                )
```
And remove the previous `message` declaration?